### DISCUSSION TITLE
How do I convert unity navemesh file to dtNavMesh?

### DISCUSSION THREAD

**Discussion by awakning** [recastnavigation/recastnavigation#564] [Q&A] [answered]
1. Bake navmesh using the dungeon .obj of RecastDemo in Unity
2. Save unity navmash as .obj ( http://wiki.unity3d.com/index.php?title=ObjExporter)
3. Load 2.obj using RecastDemo
![image](https://user-images.githubusercontent.com/84365226/118593201-3e42f700-b7e2-11eb-99e9-8541337c6942.png)

How do I convert unity navemesh .obj file to dtNavMesh?

Please help me

**Comment by grahamboree** [comment, marked as answer]
There's not any built-in way to build a dtNavMesh from an arbitrary obj, as there's more data in a dtNavMesh than just the mesh geometry.  It is possible you could write an importer for your obj that injects this data, but it's not a workflow recast supports.
